docs/tutorial/working-with-instancegroups.md

```diff
@@ -234,6 +234,29 @@ spec:
   rootVolumeIops: 200
 ```
 
+## Encrypting the root volume
+
+As of kOps 1.19.0 you can encrypt the root volume  _(note, presently confined to AWS)_ via the instancegroup specification.
+
+```YAML
+metadata:
+  name: nodes
+spec:
+  ...
+  role: Node
+  rootVolumeSize: 200
+  rootVolumeEncryption: true
+  rootVolumeEncryptionKey: arn:aws:kms:us-east-1:012345678910:key/1234abcd-12ab-34cd-56ef-1234567890ab
+```
+
+In the above example the encryption key is optional. The default key for EBS encryption is used when not specified.
+The format of encryption key can be any of the following:
+
+* key ID: `1234abcd-12ab-34cd-56ef-1234567890ab`
+* key alias: `alias/ExampleAlias`
+* key ARN: `arn:aws:kms:us-east-1:012345678910:key/1234abcd-12ab-34cd-56ef-1234567890ab`
+* alias ARN: `arn:aws:kms:us-east-1:012345678910:alias/ExampleAlias`
```

Comment:
Rather than copying this information from AWS can we just link to the AWS docs that describe these formats?

Reply:
I could not find any linkable docs. I think would be better to keep this as is than link a page where people need to search for kms-key-id.

Reply:
Heres a link we can use that takes the user right to the relevant information:

https://docs.aws.amazon.com/kms/latest/developerguide/concepts.html#key-id

Reply:
Ah thanks. I was looking for something related to EBS encryption.

Reply:
I don't see any changes since your comment, did you forget to push the new commit?

Reply:
Thanks for noticing. Done.